### PR TITLE
{bio}[foss/2016b] RNA-SeQC v1.1.8

### PR DESCRIPTION
**Pull request by SethosII**
(created using `eb --new-pr`)

I'm not sure about the toolchain. RNA-SeQC itself is just a JAR but BWA needs a toolchain. How should I express this in the EasyConfig.

### CODE REVIEW

File: easybuild/easyconfigs/r/RNA-SeQC/RNA-SeQC-1.1.8-foss-2016b-Java-1.8.0_121.eb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+
+source_urls = ['https://data.broadinstitute.org/cancer/cga/tools/rnaseqc/']
+sources = ['%(name)s_v%(version)s.jar']
+
+java = 'Java'
+javaver = '1.8.0_121'
+versionsuffix = '-%s-%s' % (java, javaver)
+dependencies = [(java, javaver)]
```
Comment (boegel) [inline easybuild/easyconfigs/r/RNA-SeQC/RNA-SeQC-1.1.8-foss-2016b-Java-1.8.0_121.eb]: Please use the `%(javaver)s` template (and move up the `versionsuffix = ...` line below `version = ...`

```python
version = '1.1.8'
versionsuffix = '-Java-%(javaver)s'
...
dependencies = [('Java', '1.8.0_121')]
```

**Comment by boegel** [comment]
@SethosII The approach you use is the right one imho, that's also how we do it in other similar situations.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
gligar02.gligar.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/66ab70cb835255bd136938e066a5af46 for a full test report.

File: easybuild/easyconfigs/r/RNA-SeQC/RNA-SeQC-1.1.8-foss-2016b-Java-1.8.0_121.eb (hunk 1 above)
Comment (SethosII) [reply easybuild/easyconfigs/r/RNA-SeQC/RNA-SeQC-1.1.8-foss-2016b-Java-1.8.0_121.eb]: @boegel I changed this.

**Comment by boegel** [comment]
Going in, thanks @SethosII!

**Pull request merged**